flink-addons/flink-streaming/flink-streaming-core/pom.xml

```diff
@@ -48,6 +48,12 @@ under the License.
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
+        
+        <dependency>
+            <groupId>org.apache.sling</groupId>
+            <artifactId>org.apache.sling.commons.json</artifactId>
+            <version>2.0.6</version>
+        </dependency>
```

Comment:
This is already a dependency of the flink-connectors.
It might be good at some point to refactor the batch JSON plan generator to use some actual JSON parser to make it look less messy